cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -893,6 +893,14 @@
     </description>
   </property>
 
+  <property>
+    <name>data.storage.sql.scan.size.rows</name>
+    <value>100</value>
+    <description>
+      The fetch size for database reads from Postgres sql.
```

Comment:
nit: update the description to reflect the unit is row

Reply:
Done